### PR TITLE
feat(TBD-6571) Add Azure support for MapR 5.2

### PR DESCRIPTION
**Pull request by yu-talend**

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java [hunk 1]
```diff
@@ -400,12 +411,12 @@ public SparkStreamingKafkaVersion getSparkStreamingKafkaVersion(ESparkVersion sp
...
     @Override
     public boolean doSupportAzureBlobStorage() {
-        return false;
+        return true;
     }
 
     @Override
     public boolean doSupportAzureDataLakeStorage() {
-        return false;
+        return true;
```
Comment (kcoepeau) [inline main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java]: I think ADLS is supported on Hadoop 2.8+.
Dependencies for ADLS were not added and starts at 2.8 version.

File: main/plugins/org.talend.hadoop.distribution.mapr600/src/org/talend/hadoop/distribution/mapr600/MapR600Distribution.java [hunk 2]
```diff
@@ -461,7 +461,7 @@ public boolean doSupportAzureBlobStorage() {
     // the changes backported to all earlier versions
     @Override
     public boolean doSupportAzureDataLakeStorage() {
-        return false;
+        return true;
```
Comment (kcoepeau) [inline main/plugins/org.talend.hadoop.distribution.mapr600/src/org/talend/hadoop/distribution/mapr600/MapR600Distribution.java]: Same as above for MapR 5.2
MapR 6.0 uses also Hadoop 2.7

File: main/plugins/org.talend.hadoop.distribution.mapr600/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -8,3 +8,4 @@ Service-Component: OSGI-INF/mapr600.xml
 Require-Bundle: org.talend.core.runtime,
  org.talend.hadoop.distribution
 Bundle-ActivationPolicy: lazy
+Export-Package: org.talend.hadoop.distribution.mapr600
```
Comment (kcoepeau) [inline main/plugins/org.talend.hadoop.distribution.mapr600/META-INF/MANIFEST.MF]: Is this change useful ? 
This is not present in other distributions.

File: main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java (hunk 1 above)
Comment (yu-talend) [reply main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java]: Support for ADLS doesn't depend on version of Hadoop. But I change it back for this moment as didn't test on it.

**Comment by kcoepeau** [comment]
ADLS still as a dependency on hadoop-common 2.8+ but not under. (except for CDH distributions)

File: main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java [hunk 4]
```diff
@@ -400,7 +411,7 @@ public SparkStreamingKafkaVersion getSparkStreamingKafkaVersion(ESparkVersion sp
     // the changes backported to all earlier versions
```
Comment (kcoepeau) [inline main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java]: I think comment is not necessary anymore

File: main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java [hunk 5]
```diff
@@ -400,7 +411,7 @@ public SparkStreamingKafkaVersion getSparkStreamingKafkaVersion(ESparkVersion sp
...
     @Override
     public boolean doSupportAzureBlobStorage() {
-        return false;
+        return true;
     }
 
     @Override
```
Comment (kcoepeau) [inline main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java]: Override not needed as false is the default

Comment (yu-talend) [reply main/plugins/org.talend.hadoop.distribution.mapr520/src/main/java/org/talend/hadoop/distribution/mapr520/MapR520Distribution.java]: Yes !

**Pull request merged**